frontend/src/components/Login/Login.tsx

```diff
@@ -113,12 +113,17 @@ function Login() {
                 </Typography>
 
                 <Snackbar
-                  anchorOrigin={{ vertical: 'bottom', horizontal: 'right' }}
+                  anchorOrigin={{ vertical: 'top', horizontal: 'right' }}
+                  sx={{ zIndex: 10000 }}
                   open={!!notification}
                   autoHideDuration={3000}
                   onClose={handleCloseNotification}
                 >
-                  <Alert onClose={handleCloseNotification} severity="error">
+                  <Alert
+                    onClose={handleCloseNotification}
+                    severity="error"
+                    sx={{ mt: '1vh' }}
+                  >
                     {notification}
                   </Alert>
                 </Snackbar>

```

Comment:
I think you should create a universal component like StyledSnackBar or something and reuse it in all HOCs, so styles won't repeat in each usage.

Just reinvoke it with the needed type and message in Login, Registration and ForgotPassword